### DISCUSSION TITLE
Restore pretty blue Win11 Start icon? (Or make a 3rd style that simply restores Toolbars to Windows 11 & doesn't change anything else)

### DISCUSSION THREAD

**Discussion by wad11656** [valinet/ExplorerPatcher#2869] [Q&A]
In Windows 10 taskbar style, how can I use the OS' default pretty blue Win11 Start menu button? It's a flat ugly black. Win11's departure from the flat, monotone "metro" style is a big plus...(Also restoring Windows 11's default rounded cute search bar would be nice...The only reason I'm using the Windows 10 style is to enable Toolbars...If there was a 3rd style that simply enabled Toolbars with everything else remaining the same Windows 11 defaults, that'd be perfect.)

**Comment by pyrates999** [comment]
Open up EP Properties application, go to start menu.  Start button style might be what you're looking for: https://github.com/valinet/ExplorerPatcher/wiki/All-features#start-button-style

**Reply by wad11656** [reply]
that is the exact "flat ugly black" icon i was talking about......................... I just want the default blue windows 11 start icon (without needing to install yet another 3rd-party app).

i proposed a 3rd taskbar style that simply has Toolbars enabled, since it sounds like future Win11 updates might break the Win10 taskbar functionality of this app:

https://github.com/valinet/ExplorerPatcher/discussions/2586#discussioncomment-8523695

...but maintaining a 3rd Taskbar style that *only* restores Toolbar functionality might be easier to maintain and keep stable/functional, even through Win11 updates that break the "Windows 10" Taskbar style

...So, even in the event that the Windows 10 Taskbar style has to go, at least we'd be left with a Taskbar Style that enables Toolbars in Windows 11

that's my idea/dream, anyway.

StartAllBack does exactly what I want (simply enables Toolbars without visibly changing anything else in the Taskbar) and sounds more stable, but costs money

**Reply by pyrates999** [reply]
I think software is worth paying for if it full fills a feature you want.

**Reply by wad11656** [reply]
Yes it would be a no-brainer if it was something I personally cared about. It's for hundreds of employees and StartAllBack licenses would cost us into the thousands. So of course my boss would be happier with me if I found a free & reliable solution to enable Toolbars in Win11. I have no idea how this program is coded but I just wondered if it were possible to isolate the Toolbar-enable functionality, which I imagine would be more reliable and resistant to crashes caused by future Win11 updates, since it's just a single feature to maintain. But of course I have no idea--just speculating and hoping. Yeah whatever. Sounds like we're paying thousands for StartAllBack

**Reply by pyrates999** [reply]
Look at open shell if you want a free start menu replacement.  As for toolbars, EP is open source.  So you can spend some programming time isolating the feature if you want.

**Reply by pyrates999** [reply]
Since you also say this is for a business, support another business and pay for the licenses for StartAllBack where you'll get tech support and other benefits.

**Reply by haydengriz** [reply]
Screw Windows 11, just because it looks “kind of” like aero doesn’t make it aero

**Reply by Amrsatrio** [reply]
The problem here with adding the toolbars to the new taskbar is that I still have to find out *how* to integrate it to the new taskbar. Because the new taskbar's code is considerably more complex compared to the old one. Unless someone is willing to give me knowledge on the new taskbar's inner workings.

TLDR: I know how the toolbars work, but don't know yet how to integrate.

**Reply by wad11656** [reply]
@Amrsatrio - I see, no problem. Thanks! I finally was able to confirm that StartIsBack business licenses should also work for StartAllBack so I'll just need to convince my boss to purchase them:

https://msfn.org/board/topic/183013-startallback-for-windows-11/?do=findComment&comment=1205468
